app/assets/javascripts/locales/i18n.js

```diff
@@ -323,7 +323,7 @@ I18n.enableVerboseLocalization = function() {
     if (!current) {
       current = keys[scope] = ++counter;
       var message = "Translation #" + current + ": " + scope;
-      if (!_.isEmpty(value)) {
+      if (value && Object.keys(value).length > 0) {
```

Comment:
Reading this, it seems like you are aware of the difference between the two isEmpty